### ISSUE TITLE
[a11y] Slash - disabled attribute doesn't have an impact on input's design

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/design-system#980]
When we set the disabled attribute for the file input, the input is indeed disabled but there's no modification of the input's design.

Actual : 
![Image](https://github.com/user-attachments/assets/ebcf4354-8013-4fbc-8eb8-72cd4871f6d8)

Expected : 
![Image](https://github.com/user-attachments/assets/b015784c-eb27-4ad5-9c64-0153fa61caed)

**Comment by MartinWeb** [comment]
One fix had been attempted on react-toolkit but it was never merged. We can inspire ourself on it
https://github.com/AxaFrance/react-toolkit/pull/1052

**Comment by BenzakourOmar** [comment]
@JLou je la prend!

**Comment by MartinWeb** [comment]
Vu avec @adrien-dos on ne gère pas le mode désactivé sur le composant File. Je clôture
